### ISSUE TITLE
Problem with ribbon 0.8.0

### ISSUE THREAD

**Issue by bowsersenior** [matheusmoreira/acclaim#5]
Hi, I ran into an issue with acclaim and ribbon 0.8.0. I get this error when I try to run the simple examples from the README:

```
acclaim-0.5.1/lib/acclaim/option/parser.rb:166:in `parse_values!': undefined method `wrap' for Ribbon:Class (NoMethodError)
```

I switched to ribbon version 0.7.0 and the error went away. I noticed that `Ribbon::Raw` replaced `Ribbon::Wrapper` in 0.8.0 so that looks like a potential cause. I couldn't find where wrap was being called in ribbon or acclaim though.

Appreciate your help and thanks for your great work on acclaim and ribbon!

-Mani

**Comment by matheusmoreira** [comment]
Indeed, that method has been removed. I made a mistake when I designed the ribbon API – general-purpose methods are often needed, and yet featureless ribbons were the default.

I refactored it, and released ribbon 0.8.0. Since I changed the public API, it is incompatible with acclaim 0.5.1, which is the most recent released version of acclaim. In your case, the error is raised on [this line](https://github.com/matheusmoreira/acclaim/blob/369d49f28ad7d00ca042c9055d5752d87c256a97/lib/acclaim/option/parser.rb#L166).

[I've already updated the codebase to reflect the change](https://github.com/matheusmoreira/acclaim/commit/26ad4dab1fa5cbb08307e65c67268f181093cc6a), but these adaptations will only be included in version 0.6.0. Since these projects are still in the development phase, I haven't specified versions in their gem specifications, which is why you got an incompatible version of ribbon.

Thanks for reporting this, and I apologize for the inconvenience. Please use the older version of ribbon for now, or try using acclaim directly from the git repository. Next time something like this happens, I'll issue a minor release to fix the problem quickly, instead of letting commits pile up.

By the way, the README is outdated as well, it does not cover some new features. I plan on writing a new one from scratch before the new version is released. I haven't been able to work on acclaim in these past weeks because I'm taking driving lessons, but I'll probably get back on it soon.

**Comment by bowsersenior** [comment]
That is very good info. Thanks for taking the time to explain in detail. I've started using acclaim on [gauntlt](https://github.com/thegauntlet/gauntlt) and it is by far the best option parser I've seen.

Good luck on the driving lessons!
